lib/travis/build/script/csharp.rb

```diff
@@ -188,7 +188,11 @@ def mono_osx_url
         end
 
         def dotnet_osx_url
-          return "https://dotnetcli.azureedge.net/dotnet/preview/Installers/#{config[:dotnet]}/dotnet-dev-osx-x64.#{config[:dotnet]}.pkg"
+          if config[:dotnet].include? "-rc"
```

Comment:
I'd invert the condition, i.e. check for `preview` as presumably the RTM version will also use the new URL and then we don't need to change again.